.travis-build-without-test.sh

```diff
@@ -14,6 +14,28 @@ export CHECKERFRAMEWORK=`readlink -f ${CHECKERFRAMEWORK:-../checker-framework}`
 
 export PATH=$AFU/scripts:$JAVA_HOME/bin:$PATH
 
+git -C /tmp/plume-scripts pull > /dev/null 2>&1 \
```

Comment:
Instead of error-prone, can we use javac.jar from eisop/jsr308-langtools?  That jar is already required to build?  Also, dependencies are already committed to the repository in `annotation-tools/annotation-file-utilities/lib`.  Would it make sense to commit `javac.jar` and `javac-jdk.jar` from  eisop/jsr308-langtools, too?

Reply:
I've changed the scripts to use error-prone javac, but now the checker-framework test `./gradlew wholeProgramInferenceTests` fail:
https://travis-ci.org/smillst/annotation-tools/jobs/569086906#L6086

(I also get this error locally.)  I'll look into, but let me know if you have any insight.